src/components/Card.js

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import "../scss/components/Card-mobile.scss";
+import "../scss/components/Card-tablet.scss";
+import "../scss/components/Card-desktop.scss";
+
+export default class Card extends React.Component {
+  state = {
+    products: this.props.data.products,
+  };
+
+  selectNormalCard = (index) => {
+    if (
+      this.state.products[index].isActive === false &&
+      this.state.products[index].isError === false
+    ) {
+      this.setState((state) => {
+        return (state.products[index].isActive = true);
+      });
+    } else {
+      this.setState((state) => {
+        return (state.products[index].isActive = false);
+      });
+    }
+  };
+
+  selectErrorCard = (index) => {
+    if (
+      this.state.products[index].isActive === false &&
+      this.state.products[index].isError === true
+    ) {
+      this.setState((state) => {
+        return (state.products[index].isActive = true);
+      });
+    } else {
+      this.setState((state) => {
+        return (state.products[index].isActive = false);
+      });
+    }
+  };
+
+  productCard = () =>
+    this.state.products.map((i, index) => (
+      <section
+        key={i.id}
+        className={
+          // IF THE BOX IS ACTIVE, ENABLE AND WITHOUT ERROR:
+          this.state.products[index].isActive &&
+          this.state.products[index].boxEnable &&
+          !this.state.products[index].isError
+            ? "product-card product-card--active"
+            : // IF THE BOX IS DISABLED:
+            "product-card" && !this.state.products[index].boxEnable
+            ? "product-card product-card--disabled"
+            : "product-card" &&
+              // IF THE BOX IS ACTIVE, ENABLE AND WITH ERROR:
+              (this.state.products[index].isActive &&
+              this.state.products[index].boxEnable &&
+              this.state.products[index].isError
+                ? "product-card product-card--active-error"
+                : "product-card")
+        }
+        onClick={
+          (this.state.products[index].boxEnable &&
+          !this.state.products[index].isError
+            ? () => this.selectNormalCard(index)
+            : console.log()) ||
+          (this.state.products[index].boxEnable &&
+          this.state.products[index].isError
+            ? () => this.selectErrorCard(index)
+            : console.log())
+        }
+      >
+        <picture className="product-card__picture picture">
```

Comment:
```suggestion
        <picture className="product-card__picture">
```
W kilku miejscach masz niepotrzebne MIX-y BEM-owe. Skoro `picture` nie swoich stylów, żadnych elementów - dosłownie nic, to znaczy, że nie jest Ci potrzebny w ogóle taki blok.

https://en.bem.info/methodology/key-concepts/#mix
https://en.bem.info/methodology/css/#mixes